requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
+ratelimiter
+pandas
```

Comment:
Are you alright with my heavy reliance on pandas? I can walk you through some things if you'd like! It just cuts down ~20-30 lines of code, but while it's more clear to me, I recognize it might not be useful to you

Reply:
seems fine for the parts where data actually needs to be manipulated, but isn't necessary for purely reading/writing CSVs imo. in a different context like a real CI build I might be concerned about the impact on build times because it has C extensions and such (and idk how big it is) but i don't think that matters here.

Reply:
Let me think about it. It feels like a heavy dependency that we don't really need, but I'm ok with it for the time being (and maybe it's a chance for me to start getting more familiar with it).